build.gradle

```diff
@@ -20,8 +20,10 @@ allprojects {
     apply plugin: 'checkstyle'
     apply plugin: 'com.diffplug.spotless'
     apply plugin: 'com.github.jk1.dependency-license-report'
+    apply plugin: 'maven-publish'
 
     group = 'org.opensearch.dataprepper'
+    version = '2.7.0-SNAPSHOT'
```

Comment:
This value already comes from the `gradle.properties`. Let's use only that one as the source of truth.

Reply:
Done. I'm still learning what Gradle plumbs in and doesn't plumb in.